### DISCUSSION TITLE
Embed GitHub issue forms

### DISCUSSION THREAD

**Discussion by brettcurtis** [GitbookIO/community#256] [Feature Requests]
It would be super cool to embed GitHub issue forms in Gitbook. Ideally, they would keep the look and feel of the Gitbook site to give users a streamlined experience.

Example: https://docs.osinfra.io/product-guides/google-cloud-platform/landing-zone#service-interfaces 
Issue form: [example](https://github.com/osinfra-io/google-cloud-shared-resources/issues/new?assignees=&labels=enhancement%2Cgood+first+issue&projects=&template=add-update-k8s-networking-resources.yml&title=%F0%9F%94%A9+Add+or+update+Kubernetes+networking+resources)

**Comment by addisonschultz** [comment]
Great idea! I can definitely see how that could be useful. After digging around a little, it doesn't look like embedding an issue form natively is supported - that being said, you could probably create a simple form using GitBook's integration platform, that sends a request to GitHub's API to submit an issue this way - something thats a bit more custom and needs to be built, but should be possible in some way!

If you're interested in diving in, I would suggest starting with our developer docs: https://developer.gitbook.com/

**Reply by brettcurtis** [reply]
Thanks, this might be a bit out of my ballpark, but I'll definitely check it out.
